# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../shadow/login_defs.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./etc/login.defs.linux.orig	Fri Jan  5 15:33:49 2001
+++ ./etc/login.defs.linux	Fri Jan  5 15:38:52 2001
@@ -299,7 +299,7 @@
 # Set to "no" if you need to copy encrypted passwords to other systems
 # which don't understand the new algorithm.  Default is "no".
 #
-#MD5_CRYPT_ENAB	no
+MD5_CRYPT_ENAB	yes
 
 #
 # List of groups to add to the user's supplementary group set
@@ -357,7 +357,7 @@
 # Setting GETPASS_ASTERISKS to -1 reverts to the traditional getpass()
 # without any new features.  This is the default.
 #
-#GETPASS_ASTERISKS 1
+GETPASS_ASTERISKS 3
 
 #
 # Enable setting of the umask group bits to be the same as owner bits
